src/lib/darknode/utils/feesUtils.ts

```diff
@@ -59,7 +59,10 @@ export const getMinimumAmountForToken = (
     }
     const { minimumAmount } = data;
     const fee = getClaimFeeForToken(symbol, blockState);
-    return fee.plus(minimumAmount);
+    const feeDivisor = AllTokenDetails.get(Token[symbol])?.feeDivisor;
+    return feeDivisor ?
+        fee.plus(minimumAmount).div(new BigNumber(Math.pow(10, feeDivisor))) :
```

Comment:
This can be a bit tidier as `fee.plus(minimumAmount).shiftedBy(-feeDivisor)` (other places in the codebase use .div/.pow as well though)